instant-xml-macros/src/lib.rs

```diff
@@ -22,15 +25,57 @@ fn retrieve_default_namespace(input: &DeriveInput) -> Option<String> {
         };
 
         if list.path.get_ident()? == "namespace" {
-            if let NestedMeta::Lit(Lit::Str(v)) = list.nested.first()? {
-                return Some(v.value());
-            }
+            return Some(list.nested.clone());
```

Comment:
Why do we need to `clone()` here?

Reply:
Not relevant anymore, it was not needed, good catch though, thanks.